### DISCUSSION TITLE
Newbie questions

### DISCUSSION THREAD

**Discussion by lewist57** [alicevision/Meshroom#2287] [Q&A]
1) what is the relative time increase to process a project on a Windows PC without a Nivida gpu? 2x, 5x, 10x?
2) since processing can take significant time, sometimes hours, how does one determine that an individual process step is "stuck" as opposed to "working"? If a process cannot complete,  could the status color change from orange to red?
3) would it be helpful to mask out unwanted details prior to processing - in other words remove all background leaving only the object to be processed in the photos?

**Comment by natowi** [comment]
1. it can be faster but is less detailed
2. take a look in the log tab of the node, it should list the current step being worked on
3. it depends. Sometimes you need the background as reference. Masking the background can speed up the meshing in some cases, as the background is not reconstructed

**Comment by lewist57** [comment]
Thank you for your response. I have found a version of Meshroom that works with my OpenCL AMD gpu on Github. Overall Meshroom is an amazing project, job well done!

Sent via the Samsung Galaxy S20 FE 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
